tests/baselines/reference/objectCreate.js

```diff
@@ -0,0 +1,28 @@
+//// [objectCreate.ts]
+
+declare var union: null | { a: number, b: string };
+
+var n = Object.create(null);             // object
```

Comment:
in an input `{}` is not the same as `object`. object is more specific than `{}`. in here it does not matter. i used `object` to avoid having a type of `{} | object`.